app/preview.py

```diff
@@ -77,7 +78,8 @@ def view_letter_template(filetype):
         values=json['values'] or None,
         contact_block=json['letter_contact_block'],
         # we get the images of our local server to keep network topography clean, which is just http://localhost:6013
-        admin_base_url='http://localhost:6013'
+        admin_base_url='http://localhost:6013',
+        logo_file_name=current_app.config['LOGO_FILENAMES'][json['dvla_org_id']],
```

Comment:
reckon we should `abort(400)` if this isn't specified (rather than 500 KeyError)